src/index.js

```diff
@@ -1 +1,12 @@
-import './style.css';
\ No newline at end of file
+import './style.css';
+import { createGame, addGame, refreshGame } from './modules/createGame.js';
+
+const refresh = document.querySelector('.refresh');
+const submit = document.querySelector('.submit');
+
+createGame();
+
```

Comment:
**[Optional but Highly Recommended]**

- [ ] I have experienced that every time your page reloads, it creates a new game id which is applicable in some cases but not in this project. It would be nice if you use only one id every time your page reloads because the leaderboard contains data from all the users if it will create a new id every time then it would delete the previous data 😨 which doesn't make any sense 🤔 Kindly make this change 👍 You don't need to make a call for the game id every time. Just make a call from postman or from your application just one time and save the game id in your application and use it 😊

Reply:
then should i store it in the local storage @HAMMAS-SALEEM

Reply:
Hey @gaks1 👋

No dear i am not saying you should store it in localStorage because if you store it in the local storage then this id won't be available for the other users of your application and for them your application will create a new id and after that new id they won't be able to see your data 🤔

I am saying that you need only one id for you and for all the users of your application. So it would be nice if you add that id in an variable in your javascript file and use it 😊